packages/governance/src/contractHelper.js

```diff
@@ -42,25 +43,44 @@ const facetHelpers = (zcf, paramManager) => {
 
   const { electionManager } = terms;
 
+  const commonPublicMethods = {
+    getSubscription: () => paramManager.getSubscription(),
+    getContractGovernor: () => electionManager,
+    getGovernedParams: () => paramManager.getParams(),
+  };
+
   /**
+   * Add required methods to publicFacet
+   *
    * @template {{}} PF public facet
    * @param {PF} originalPublicFacet
    * @returns {GovernedPublicFacet<PF>}
    */
   const augmentPublicFacet = originalPublicFacet => {
     return Far('publicFacet', {
       ...originalPublicFacet,
-      getSubscription: () => paramManager.getSubscription(),
-      getContractGovernor: () => electionManager,
-      getGovernedParams: () => paramManager.getParams(),
+      ...commonPublicMethods,
       ...typedAccessors,
     });
   };
 
+  /**
+   * Add required methods to publicFacet, for a virtual/durable contract
+   *
+   * @param {{}} originalPublicFacet
+   */
+  const augmentVirtualPublicFacet = originalPublicFacet => {
+    return Far('publicFacet', {
+      ...originalPublicFacet,
+      ...commonPublicMethods,
+      ...objectMap(typedAccessors, dropContext),
```

Comment:
`dropContext` name confused me. I read it as the opposite, taking a function with context and making it one without. I see the logic now in that it wraps in functions that drop context before passing it on.

@erights I think as a verb it should describe what it does to the underlying object to make it into the return type. What it's returning is a function that drops context, but it isn't itself dropping context. So perhaps `ignoreContext`.

Not a blocker and maybe out of scope.

Reply:
I like `ignoreContext`.

I do think it is out of scope for this PR. But please let's do this renaming soon.

Reply:
I'll take that on after this lands.